src/towncrier/check.py

```diff
@@ -57,15 +57,28 @@ def _get_default_compare_branch(branches: Container[str]) -> str | None:
     metavar="FILE_PATH",
     help=config_option_help,
 )
-def _main(compare_with: str | None, directory: str | None, config: str | None) -> None:
+@click.option(
+    "--staged",
+    "staged",
+    is_flag=True,
+    default=False,
+    metavar="STAGED",
+    help="include staged files as part of the branch checked in the --compare-with",
```

Comment:
Let's capitalize this like the others:

```suggestion
    help="Include staged files as part of the branch checked in the --compare-with",
```